bin/desi_tile_qa

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/env python
+#
+# See top-level LICENSE.rst file for Copyright information
+#
+# -*- coding: utf-8 -*-
+
+"""
+This script computes QA scores per exposure, after the cframe are done
+"""
+
+
+import os,sys
+import argparse
+import glob
+import numpy as np
+import multiprocessing
+from astropy.table import Table
+import fitsio
+
+from desiutil.log import get_logger
+from desispec.io import specprod_root,findfile,read_tile_qa,write_tile_qa
+from desispec.tile_qa import compute_tile_qa
+from desispec.util import parse_int_args
+
+from desispec.tile_qa_plot import make_tile_qa_plot
+
+
+def parse(options=None):
+    parser = argparse.ArgumentParser(
+                description="Calculate tile QA")
+    parser.add_argument('-o','--outfile', type=str, default=None, required=False,
+                        help = 'Output summary file')
+    parser.add_argument('--recompute', action = 'store_true',
+                        help = 'recompute')
+    parser.add_argument('--prod', type = str, default = None, required=False,
+                        help = 'Path to input reduction, e.g. /global/cfs/cdirs/desi/spectro/redux/blanc/,  or simply prod version, like blanc, but requires env. variable DESI_SPECTRO_REDUX. Default is $DESI_SPECTRO_REDUX/$SPECPROD.')
+    parser.add_argument('--exposure-qa-dir', type = str, default = None, required=False,
+                        help = 'Path to exposure qa directory, default is the input prod directory')
+    parser.add_argument('--outdir', type = str, default = None, required=False,
+                        help = 'Path to ouput directory, default is the input prod directory')
```

Comment:
Please clarify in help string whether output goes directly into outdir without any sub-hierarchy, or into outdir/tiles/cumulative/TILEID/NIGHT/, or something else.